### PR TITLE
Adding data in resolution response, adding test stats in status endpoint

### PR DESCRIPTION
**Pull request by opishcheiko**
- I need this data in resolution response to try and fix that bug on the frontend
- adding stats to endpoint where tests can be retrieved by status, so it has all the same props as a normal full endpoint (makes sense to show how many tests failed on failed tests page)

### CODE REVIEW

**Comment by opishcheiko** [comment]
@Jnegrier
 hi Juan, can you please have a look if these small changes are ok?

File: app.py [hunk 1]
```diff
@@ -583,7 +583,11 @@ def update_test_history_resolution():
         params.get("test_history_id"), params.get("test_resolution")
     )
 
-    data = {"message": "Test history resolution updated successfully"}
+    data = {
+        "message": "Test history resolution updated successfully", 
+        "resolution": params.get("test_resolution"), 
+        "test_history_id": params.get("test_history_id")
```
Comment (Jnegrier) [inline app.py:589]: It's a bit strange to return the same parameters you're receiving 😸 I think you modify the method that update test_history in the crud file, and return the whole object that was updated https://github.com/delta-reporter/delta-core/blob/failed-tests-new-way/data/crud.py#L639 then you could save it in this piece and use it to get the resolution and test_history_id

File: app.py [hunk 2]
```diff
@@ -833,6 +843,12 @@ def get_tests_history_by_test_status_and_test_run_id(test_status_id, test_run_id
                             test_suite_history.end_datetime,
                         ),
                         "test_suite_status": test_suite_history.test_suite_status.name,
+                        "tests_total": total_count if total_count else 0,
+                        "tests_failed": failed_count if failed_count else 0,
+                        "tests_passed": passed_count if passed_count else 0,
+                        "tests_running": running_count if running_count else 0,
+                        "tests_incomplete": incomplete_count if incomplete_count else 0,
+                        "tests_skipped": skipped_count if skipped_count else 0,
```
Comment (Jnegrier) [inline app.py:851]: I wonder if these are going to work, I don't think we are using subqueries for the method getting this data 🤔

**Comment by opishcheiko** [comment]
@Jnegrier can you check again please?

**Pull request merged**